frontend/components/site/siteBuildLogs.jsx

```diff
@@ -1,23 +1,48 @@
 /* eslint-disable react/forbid-prop-types */
 
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Link, useParams } from 'react-router-dom';
 
 import { useBuildLogs } from '../../hooks';
+import LoadingIndicator from '../LoadingIndicator';
 import SiteBuildLogTable from './siteBuildLogTable';
 import DownloadBuildLogsButton from './downloadBuildLogsButton';
 import CommitSummary from './CommitSummary';
+import api from '../../util/federalistApi';
 
 export const REFRESH_INTERVAL = 15 * 1000;
 
+function buildTooOld(build) {
+  return (new Date() - new Date(build.completedAt)) > 15552000000;
```

Comment:
Perhaps save this integer as a const so people know what it means? I think its 90 days in miliseconds but i can't remember. Similar to REFRESH_INTERVAL a few lines up.